data/test/scenarios/event_handlers_in_events.cfg

```diff
@@ -266,6 +382,15 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [event][insert_tag]
```

Comment:
It's a bug in the test rather than the documentation, but this test can pass regardless of the filter, because `first_time_only` defaults to `yes`.

Reply:
It should have `first_time_only=no`, not `yes`; however I've just tested and the filter works both ways. Explanation:

```
##
# Actions:
# Define a post_advance event which includes a filter created via [insert_tag]
# Two units are created, exactly one of which matches the filter
# Both units are advanced a level
##
# Expected end state:
# The event was triggered exactly once
##
```

Reply:
Updated.